archive/html_css/semantic_html.md

```diff
@@ -98,4 +98,6 @@ This section contains helpful links to other content. It isn’t required, so co
 
 * [NVAccess](https://www.nvaccess.org/download/) provides a download of the NVDA screen reader, one of the more popular (and free!) screen readers available, though it's only available for the Windows OS (this is the **only** time where we're saying it's okay to use Windows in the scope of this curriculum). If you're using a macOS device, you should have the VoiceOver screen reader available to you by default. While the lessons in this section will provide you with what a screen reader will generally announce, it can be invaluable to start using a screen reader yourself to test out your projects and see how they might be perceived.
 * [Screen Reader Basics: VoiceOver](https://www.youtube.com/watch?v=5R-6WvAihms&list=PLNYkxOF6rcICWx0C9LVWWVqvHlYJyqw7g&index=8) and [Screen Reader Basics: NVDA](https://www.youtube.com/watch?v=Jao3s_CwdRU&list=PLNYkxOF6rcICWx0C9LVWWVqvHlYJyqw7g&index=9) go over some basics for getting started with both brands of screen readers. Both videos are worth watching regardless of which screen reader you choose/is available to you, as they contain some universal information as well.
+* ]Screen reader Basics: ChromeVox](https://www.youtube.com/watch?v=fpbIsN31hLM) Is a good place to start learning about the screen reader on ChromeBooks.
+* [Screen reader Basics: Orca](https://www.youtube.com/watch?v=UI76P-KPZec) is where you can learn about the screen reader on Linux desktops.
 * [ARIA Landmarks Example](https://www.w3.org/TR/wai-aria-practices/examples/landmarks/HTML5.html) lists the native HTML elements that define landmark regions as well as what their role is. If you check this resource out, be sure to enable the button toggles at the top of the page to see how the page is sectioned by landmarks and how it uses headings.
```

Comment:
I would undo the changes in this file, the archive directory is essentially just a graveyard for content once we've deprecated it for one reason or another.